spec/index.bs

```diff
@@ -5390,13 +5390,19 @@ dictionary GPUExternalTextureDescriptor
     </pre>
 </div>
 
-### Sampling External Textures ### {#external-texture-sampling}
+## Sampling External Texture Bindings ## {#external-texture-sampling}
+
+The {{GPUBindGroupLayoutEntry/externalTexture}} binding point enables the integration of dynamic
+image sources. While primarily intended for {{GPUExternalTexture}} (video, camera), it also supports
```

Comment:
nit: in case we end up using GPUExternalTexture for things other than video in the future, it's easier if this doesn't need to stay in sync with that.
```suggestion
image sources. While primarily intended for {{GPUExternalTexture}}, it also supports
```